hs_file_types/models/base.py

```diff
@@ -998,33 +992,17 @@ def supports_unzip(self):
     @property
     def aggregation_name(self):
```

Comment:
@pkdash - I seem to have butchered this method when trying to merge single file and folder aggregations.  If you get a chance, would you mind taking a look?  Tests are failing in `hs_file_types/tests/test_view_functions.py`

Reply:
@sblack-usu I ran the same test locally and it didn't fail. I will wait for Jenkins to run the tests again.

Reply:
Never mind. It actually fails. After I updated my local branch with remote, it fails now. Will fix it.

Reply:
@pkdash - do you agree using folder here no longer makes sense?   I was surprised you put it back to a folder, but wanted to make sure we're on the same page before I rework this.

Reply:
I've fixed at least some of the tests by allowing fileset to use folder for the aggregation_name.  The full suite is running now.  Still sorting out a lot of this in my head, but I think this will be okay.